### PR TITLE
Issue #121 Maven servlet returns JSON when JSON requested

### PR DESCRIPTION
**Pull request by doggy-dev**
Adds the ability to return Josn formatted repository information as requested in Issue #121

### CODE REVIEW

**Comment by doggy-dev** [comment]
It seem there is a problem with the eclipse target.

**Comment by ctron** [comment]
Yes, it looks like the ZIP file was deleted from Eclipse download servers. That sucks. It has to be replaced with a download location that is still available.

**Comment by ctron** [comment]
Ok, I am downloading the file now from the archive server. When you rebase your PR on the most recent master branch, then the error should be gone.

File: bundles/org.eclipse.packagedrone.repo.adapter.maven/src/org/eclipse/packagedrone/repo/adapter/maven/internal/MavenHandler.java [hunk 1]
```diff
@@ -84,7 +91,30 @@ public void handle ( final String path, final HttpServletRequest request, final
             }
 
             response.setStatus ( HttpServletResponse.SC_OK );
-            renderDir ( response, (DirectoryNode)node, path );
+
+            final String acceptHeader = request.getHeader ( "Accept" );
+            if ( acceptHeader != null && acceptHeader.contains ( "application/json" ) )
+            {
+                final GsonBuilder gsonBuilder = new GsonBuilder ();
```
Comment (ctron) [inline bundles/org.eclipse.packagedrone.repo.adapter.maven/src/org/eclipse/packagedrone/repo/adapter/maven/internal/MavenHandler.java]: Please extract this as a method like `renderDir` … e.g. `renderAsJson`

**Comment by ctron** [comment]
@doggy-dev I would like to make a release in the next few days. So if you can work on this PR in the next few days, I will wait for this. Otherwise we can add this to the next release.

**Comment by doggy-dev** [comment]
Oh sorry that I didn't see your comment earlier, I'll take care of this this week.

**Comment by doggy-dev** [comment]
Could you please merge this PR so I can rebase the #129 ?

**Pull request merged**